feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -46,19 +57,27 @@ class HomeViewModel @Inject constructor(
     fun handleEvents(event: HomeScreenEvent) = when (event) {
         is HomeScreenEvent.FeedingPointSelected -> selectFeedingPoint(event)
         is HomeScreenEvent.FeedingPointFavouriteChange -> changeFavouriteFeedingPoint(event)
-        is HomeScreenEvent.ShowWillFeedDialog -> showWillFeedDialog()
-        is HomeScreenEvent.DismissWillFeedDialog -> dismissWillFeedDialog()
-        is HomeScreenEvent.GeolocationPermissionStatusChanged -> changeGeolocationPermissionStatus(event)
+        is HomeScreenEvent.RouteEvent -> {
+            handleRouteEvent(event)
+            when (event) {
+                HomeScreenEvent.RouteEvent.FeedingRouteCancellationRequest -> fetchFeedingPoints()
+                HomeScreenEvent.RouteEvent.FeedingRouteStartRequest -> hideFeedingPointsAndSaveFeeder()
+                else -> {}
```

Comment:
Add a comment, why else block is empty

Reply:
I've actually refactor it to look like this: 
```
is HomeScreenEvent.RouteEvent -> {
            when (event) {
                HomeScreenEvent.RouteEvent.FeedingRouteCancellationRequest -> fetchFeedingPoints()
                HomeScreenEvent.RouteEvent.FeedingRouteStartRequest -> hideFeedingPointsAndSaveFeeder()
                else -> handleRouteEvent(event)
            }
        }
```

Good suggestion!